cdap-standalone/src/main/resources/cdap-site.xml

```diff
@@ -69,7 +69,7 @@
 
   <property>
     <name>app.program.spark.compat</name>
-    <value>spark2_2.11</value>
+    <value>spark3_2.12</value>
     <description>
       Spark compatibility version. Must be 'spark2_2.11' or 'spark3_2.12'. This is only
```

Comment:
nit: Please update the doc comment to remove spark2, thanks!

Reply:
Ack.